src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -117,12 +134,70 @@ const getAccountNumberSchema = (
   }
 }
 
+const getIbanNumberSchema = (
+  implicitParams: {
+    country: string
+    fiatAccountType: FiatAccountType
+  },
+  countryOverrides?: FiatAccountSchemaCountryOverrides
+): FiatAccountFormSchema<FiatAccountSchema.IBANNumber> => {
+  const overrides = countryOverrides?.[implicitParams.country]?.[FiatAccountSchema.IBANNumber]
+  //Ensure spaces/lower case don't lead to invalid input
+  const cleanInput = (input: string) => input.replace(/\s/g, '').toUpperCase()
+
+  const ibanNumberValidator = (input: string) => {
```

Comment:
it looks like for account number, we enable overrides for error string in addition to the regex. seems like it's probly worth enabling that for iban, too.